lib/shopify_theme/releases.rb

```diff
@@ -0,0 +1,57 @@
+require 'httparty'
+require 'rss'
+
+module ShopifyTheme
+  FEED_URL = 'https://github.com/Shopify/Timber/releases.atom'
+  ZIP_URL  = 'https://github.com/Shopify/Timber/archive/%s.zip'
+
+  class Releases
+    class VersionError < StandardError; end
+    Release = Struct.new(:version) do
+      def zip_url
+        ZIP_URL % version
+      end
+    end
+
+    def fetch!
+      response = HTTParty.get(FEED_URL)
+      raise "Could not retrieve feed from #{FEED_URL}" if response.code != 200
+      @feed = RSS::Parser.parse(response.body)
+    end
+
+    def all
+      @all ||= begin
+        versioned_releases.reduce({'master' => master, 'latest' => latest}) do |all, release|
```

Comment:
`master` and `latest` are elements I want to be included in the collection. I *could* just append the versioned releases and append them to an array containing the `master` and `latest` items as well.

I'm really just using a hash so I can easily grab the keys :P